### PR TITLE
Theming

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

**Review by Kmaschta — COMMENTED** [review]
Are the "primary1Color" label for example coming from the MUI documentation.. ?

File: package-lock.json [hunk 1]
```diff
@@ -1910,7 +1910,7 @@
                 },
                 "readable-stream": {
                     "version": "2.3.6",
-                    "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
+                    "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
```
Comment (Kmaschta) [inline package-lock.json]: Please keep the HTTPS here!

File: src/app/js/theme.js [hunk 2]
```diff
@@ -1,11 +1,5 @@
 export default {
-    green: {
-        primary: '#7DBD42',
-    },
-    purple: {
-        primary: '#B22F90',
-    },
-    orange: {
-        primary: '#F48022',
```
Comment (Kmaschta) [inline src/app/js/theme.js:9]: I wrote `orange.secondary` in another PR ... Can you keep it here ? :/

**Review by Kmaschta — CHANGES_REQUESTED** [review]

**Review by Kmaschta — APPROVED** [review]

File: src/app/js/theme.js (hunk 2 above)
Comment (ThieryMichel) [reply src/app/js/theme.js:9]: done

**Pull request merged**